x/warp/contract.go

```diff
@@ -264,13 +282,18 @@ func sendWarpMessage(accessibleState contract.AccessibleState, caller common.Add
 		accessibleState.GetBlockContext().Number().Uint64(),
 	)
 
+	packed, err := PackSendWarpMessageOutput(common.Hash(unsignedWarpMessage.ID()))
+	if err != nil {
```

Comment:
curious as to why we now return the packed message output? is this for verification purposes for testing? can we also update the function description?

Reply:
we have changed the function signature to return messageID as output.